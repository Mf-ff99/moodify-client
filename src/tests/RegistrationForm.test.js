import React from 'react'
import RegistrationForm from '../components/RegistrationForm'

import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('Registration Form', () => {
    it('Renders without crashing', () => {
        shallow(<RegistrationForm />)
    })
    it('renders form approriately', () => {
        const wrapper = shallow(<RegistrationForm />)
        expect(toJson(wrapper)).toMatchSnapshot()
      })
})