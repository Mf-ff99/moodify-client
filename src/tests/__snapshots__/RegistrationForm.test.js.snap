// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Registration Form renders form approriately 1`] = `
<form
  className="registration-form"
  onSubmit={[Function]}
>
  <p>
    Register and log in to continue
  </p>
  <div
    role="alert"
  />
  <div
    className="user_name"
  >
    <label
      htmlFor="RegistrationForm__user_name"
    >
      Username 
      <Required />
    </label>
    <Input
      id="RegistrationForm__user_name"
      name="user_name"
      required={true}
      type="text"
    />
  </div>
  <br />
  <div
    className="password"
  >
    <label
      htmlFor="RegistrationForm__password"
    >
      Password 
      <Required />
    </label>
    <Input
      id="RegistrationForm__password"
      name="password"
      required={true}
      type="password"
    />
  </div>
  <br />
  <Button
    type="submit"
  >
    Register
  </Button>
</form>
`;
